.git-hooks/helpers/detect-foxtrot-merge

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#usage:
+#   foxtrot-merge-detector [<branch>]
+#
+# If foxtrot merge detected for branch (current branch if no branch),
+# exit with 1.
+
+# foxtrot merges:
+# See http://bit-booster.blogspot.cz/2016/02/no-foxtrots-allowed.html
+# https://stackoverflow.com/questions/35962754/git-how-can-i-prevent-foxtrot-merges-in-my-master-branch
+# https://git-blame.blogspot.com/2012/03/fun-with-first-parent.html
+# https://stackoverflow.com/questions/35962754/how-can-i-prevent-foxtrot-merges-in-my-master-branch
+
+remoteBranch=$(git rev-parse --abbrev-ref "$1"@{u} 2>/dev/null)
+# no remote tracking branch, exit
+if [[ -z "$remoteBranch" ]]; then
+    echo "no remote tracking branch, no foxtrot merge detected"
```

Comment:
Will we actually see the non-error `echo`s in practice? My preference would be to not get notified when nothing is wrong.

Reply:
Yup, we would, and I'd generally agree, but the husky pipeline is pretty verbose already so I didn't notice them --  commented these out for now :)